code/modules/events/radiation.dm

```diff
@@ -19,9 +19,11 @@
 		for (var/pulses = pulse_amt, pulses > 0, pulses--)
 			pulseloc = pick(wormholeturfs)
 			pulse_lifespan = rand(min_pulse_lifespan,max_pulse_lifespan)
-			new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan)
+			pick(prob(90); new /obj/anomaly/radioactive_burst(pulseloc,lifespan = pulse_lifespan), prob(10); new /obj/anomaly/neutron_burst(pulseloc,lifespan = pulse_lifespan))
```

Comment:
that's not how probabilities work

Reply:
I realize this now, looking at the miraclium proc for reference